### ISSUE TITLE
How do you run regular fixmatch in your codebase

### ISSUE THREAD

**Issue by kradkahaddi** [Gank0078/FineSSL#4]
In the implementation details section of your paper, A.1 you mentioned that you use the same general experiment structure for Fixmatch and other competing methods. Where can I find the relevant sections in the codebase to run regular Fixmatch and other methods?

**Comment by Gank0078** [comment]
Hi, your can refer to https://github.com/microsoft/Semi-supervised-learning for detailed other SSL methods. Most of them only need little minor changes to the code in trainer.py.

**Comment by tian1327** [comment]
@Gank0078 Thank you for the great work! Could you please share how you run DebiasePL, specifically what hyperparameters you used for `tau` (logit adjustment) or `lambda` (weight in marginal loss)? I tried to reproduce the DebiasPL on semi-Aves dataset, but found it has no improvement over FixMatch.

I would appreciate your sharing! Thanks!
